client/src/pages/editor.css

```diff
@@ -37,3 +37,24 @@
 .editor main::-webkit-scrollbar-thumb {
   background: var(--q-color-gray-5);
 }
+
+.editor__options {
+  --q-h1-size: 13px;
+  --q-h2-size: 13px;
+  --q-h3-size: 13px;
+  --q-h4-size: 13px;
+  --q-h5-size: 13px;
+  --q-h6-size: 13px;
+}
+
+.editor__options legend {
+  border-bottom: 1px solid var(--q-color-gray-4);
+  color: var(--q-text-light-color);
+  margin-bottom: calc(var(--q-space-base) / 4);
+}
+
+
+/* make the options more compact by removing the margin around fieldsets */
+.editor__options fieldset > schema-editor-wrapper {
+  margin: 0 !important;
```

Comment:
not really as we are really specific with this selector because in main editor we change this with nesting.
I am not really happy yet with the editor styling regarding the nesting of objects/arrays. But this is something for another PR imho as it probably needs some rethinking...